importer/helper.go

```diff
@@ -237,27 +238,48 @@ func loadGenesis(db idb.IndexerDb, in io.Reader) (err error) {
 }
 
 // InitialImport imports the genesis block if needed. Returns true if the initial import occurred.
-func InitialImport(db idb.IndexerDb, genesisJSONPath string, l *log.Logger) bool {
-	state, err := db.GetImportState()
+func InitialImport(db idb.IndexerDb, genesisJSONPath string, client *algod.Client, l *log.Logger) bool {
```

Comment:
Why would you want to read from a file if you can get it from algod?

Reply:
The first import mode when Indexer was being written was all from files. Did we fully delete that?

Reply:
The legacy import command is still being used for the bash tests.